src/common/error/auth.error.ts

```diff
@@ -0,0 +1,10 @@
+export const INVALID_CREDENTIALS_ERROR = 'Invalid credentials';
+export const PASSWORD_NOT_EQUAL_ERROR = 'Passwords do not match';
+export const EMAIL_BUSY_ERROR = 'Email is busy';
+export const INVALID_TOKEN_ERROR = 'Invalid token or expired';
+export const ACCESS_RIGHTS_ERROR = 'You do not have access rights';
+export const INVALID_ID_ERROR = 'Invalid format id';
+
+export const USER_ALREADY_EXISTS_ERROR = 'User already exists';
+export const USER_EXISTS_EMAIL_ERROR = 'User with this email is exists ';
+export const USER_NOT_FOUND_ERROR = 'User not found';
```

Comment:
You can use export only one time; export const nameOfVariable = {
 USER_ALREADY_EXISTS_ERROR: "User already exists",
 USER_ALREADY_EXISTS_ERROR: "User already exists"
}